src/backend/joanie/payment/backends/lyra/__init__.py

```diff
@@ -0,0 +1,342 @@
+"""Lyra Payment Backend"""
+
+import base64
+import hashlib
+import hmac
+import json
+import logging
+from decimal import Decimal as D
+
+from django.conf import settings
+
+import requests
+from rest_framework.parsers import FormParser, JSONParser
+
+from joanie.core.models import Order
+from joanie.payment import exceptions
+from joanie.payment.backends.base import BasePaymentBackend
+from joanie.payment.models import CreditCard
+
+logger = logging.getLogger(__name__)
+
+
+class LyraBackend(BasePaymentBackend):
+    """
+    The Lyra payment backend
```

Comment:
According to settings change, I think it could be interesting to give here an example to copy/past to set environment variable for Lyra